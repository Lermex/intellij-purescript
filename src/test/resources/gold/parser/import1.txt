Purescript File
  PSASTWrapperElement(Program)
    PSASTWrapperElement(Module)
      PsiElement(module)('module')
      PsiElement(whitespace)(' ')
      PSASTWrapperElement(ModuleName)
        PSASTWrapperElement(Qualified)
          PsiElement(proper name)('A')
        PsiElement(whitespace)(' ')
      PsiElement(where)('where')
      PsiElement(whitespace)('\n\n')
      PSASTWrapperElement(NamedBinder)
        <empty list>
      PSASTWrapperElement(ImportDeclaration)
        PsiElement(import)('import')
        PsiElement(whitespace)(' ')
        PSASTWrapperElement(ImportModuleName)
          PSASTWrapperElement(Qualified)
            PsiElement(proper name)('B')