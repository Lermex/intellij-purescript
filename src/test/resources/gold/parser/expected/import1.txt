Purescript File
  PSModuleDeclarationImpl(MODULE_DECLARATION)
    PsiElement(PSTokenType.module)('module')
    PSModuleNameImpl(MODULE_NAME)
      PsiElement(PSTokenType.PROPER_NAME)('A')
    PsiElement(PSTokenType.where)('where')
  PSDeclarationsImpl(DECLARATIONS)
    PSDeclarationImpl(DECLARATION)
      PSImportDeclarationImpl(IMPORT_DECLARATION)
        PsiElement(PSTokenType.import)('import')
        PSImportNameImpl(IMPORT_NAME)
          PsiElement(PSTokenType.PROPER_NAME)('B')